encoding/thrift/doc.go

```diff
@@ -34,11 +34,12 @@
 //
 // In addition to generating code for types specified in your Thrift file,
 // this will generate two packages for each service in the file: a client
-// package and a server package.
+// package, a server package, and a test package.
 //
 // 	myservice
 // 	   |- myserviceclient
 // 	   |- myserviceserver
+// 	   |- myservicetest
```

Comment:
do we want to add more test functions in here in future? (Just want to make sure we want this to be `myservicetest` instead of `myservicemock`)